src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const { router: userRouter } = require('./routes/users.js');
+const { router: expenseRouter } = require('./routes/expenses');
+
+const userService = require('./services/users');
+const expenseService = require('./services/expenses');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  userService.getInitial();
```

Comment:
it's not clear why you name the function as `getInitial` and don't get anything here
check proper function name